src/App/Patterns/content/Forms/index.js

```diff
@@ -78,137 +129,302 @@ const HowToDesignForms = () => {
+                                    label="E-mail address"
+                                    className="ml-4 mb-4"
+                                />
                             </div>
-                            <RadioButton label="Call" name="logical-flow-example-2" id="logical-flow-example-call-2" className="mb-3"/>
+                            <RadioButton
+                                label="Call"
+                                name="logical-flow-example-2"
+                                id="logical-flow-example-call-2"
+                                className="mb-3"
+                            />
                             <div className="ml-2">
-                                <InputGroup id="logical-flow-dont-phone-example" type="text" label="Phone number" className="ml-4"/>
+                                <InputGroup
+                                    id="logical-flow-dont-phone-example"
+                                    type="text"
+                                    label="Phone number"
+                                    className="ml-4"
+                                />
                             </div>
                         </div>
                     </div>
                 </div>
             </section>
             <section>
                 <h3>Button placement</h3>
-                <p>The primary button should be left-aligned along with the left side of the form field. If additional actions are required, put the primary button first, then secondary buttons, then the link button. One exception is a multiple-step form where the “Back” secondary button sits at the very left, while the rest of the buttons are aligned to the right. In some cases, like on a <a href="">Login page</a>, buttons can be centered and set to the same width as the input fields. On mobile, the buttons are always centered, full width and stacked vertically with the primary action on top. </p> {/* TODO: Add link to Login template */}
-                <img src={`${process.env.basename}img/documentation/patterns/img-button-placement.png`} className="w-100" alt="Visualization of button placement in forms."/>
+                <p>
+          The primary button should be left-aligned along with the left side of
+          the form field. If additional actions are required, put the primary
+          button first, then secondary buttons, then the link button. One
+          exception is a multiple-step form where the “Back” secondary button
+          sits at the very left, while the rest of the buttons are aligned to
+          the right. In some cases, like on a <a href="">Login page</a>, buttons
+          can be centered and set to the same width as the input fields. On
+          mobile, the buttons are always centered, full width and stacked
+          vertically with the primary action on top.{" "}
+                </p>{" "}
+                {/* TODO: Add link to Login template */}
```

Comment:
TODO

Reply:
this is from before, it was already there